engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -228,6 +241,14 @@ private void initPipeline() {
         renderingPipeline.add(ambientOcclusionPassesNode);
         renderingPipeline.add(prePostCompositeNode);
         renderingPipeline.add(simpleBlendMaterialsNode);
+        // Post-Processing proper: tone mapping, bloom and blur passes // TODO: verify if the order of operations around here is correct
+        renderingPipeline.add(lightShaftsNode);
+        renderingPipeline.add(initialPostProcessingNode);
+        renderingPipeline.add(downSampleSceneAndUpdateExposure);
+        renderingPipeline.add(toneMappedSceneNode);
+        renderingPipeline.add(bloomPassesNode);
+        renderingPipeline.add(blurPassesNode);
+        renderingPipeline.add(finalPostProcessingNode);
```

Comment:
In light of where we are heading I'd recommend you to rename this method to "initRenderGraph" and the variable renderingPipeline to renderGraph. The renderPipeline (shorter!) will be the list of commands generated out of the graph.